docs/security.rst

```diff
@@ -160,6 +160,26 @@ look like the following.
           -H 'Cache-Control: no-cache' \
           -H "Authorization: Bearer ${ID_TOKEN}"
 
+Basic authentication
+''''''''''''''''''''
+
+Basic username password authentication is currently supported for the API.
```

Comment:
Does this authentication method always work? Are there configuration options that could prevent this authorization from working?

Reply:
I have added more comment around this topic.